contracts/Gateway/INativeTokenGateway.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
```

Comment:
We are usually using `BSD-3-Clause`

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/361/commits/8fb63cc391e405875984aba45debc493d70d652f